saylua/models/user.py

```diff
@@ -59,11 +59,14 @@ class User(db.Model):
     bio = db.Column(db.Text(), default="Hello, world!") # FIXME: This should really have a limit.
 
     def __repr__(self):
-        return '<User %r>' % self.display_name
+        return '<User %r>' % self.name()
```

Comment:
I changed it to work with user.name, but haven't touched display_name.name yet. I think I will change that one once I get multiple usernames working again.